pyserver/conf/mme_exporter.json

```diff
@@ -10,5 +10,15 @@
         "edgeName": "edge-onf-menlo",
         "tac": 17
       }
+    ],
+    "phoneTypes": [
```

Comment:
Hmmm... :) Its something which actually DPI should do. Worried about sim swapping. Was wondering if we could do based on IMEI ?

Reply:
If we can map IMSI-IMEI and MME can send IMEI to the Flask server, then we can use IMEI for this.

Reply:
@badhrinathpa Any idea?